stun/client.go

```diff
@@ -98,12 +104,22 @@ func (c *Client) Discover() (NATType, *Host, error) {
 	conn := c.conn
 	if conn == nil {
 		var laddr *net.UDPAddr
+
+		address := ""
+		if c.localIP != "" {
+			address += c.localIP
+		}
 		if c.localPort != 0 {
-			laddr, err = net.ResolveUDPAddr("udp", fmt.Sprintf(":%d", c.localPort))
+			address += strconv.Itoa(c.localPort)
```

Comment:
i think we need a `:` between ip and port

Reply:
Yes, you're right, I forgot about this